src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,17 @@
-// add styles here
+
```

Comment:
Remove this empty line from the beginning of the file and add one to the end of it